### PR TITLE
Feature/vijay playlist improvements

### PR DESCRIPTION
**Pull request by mob-dev**
- Navigate to user profile on author click in playlist screen
- Show animated playing icon in playlist screen
- Remove animations from discover & profile modules
- Remove playlists header from audio page in profile screen

### CODE REVIEW

File: lib/modules/discover/audios/discover_audios_view.dart [hunk 1]
```diff
@@ -35,19 +34,17 @@ class _DiscoverAudiosViewState
...
-              case FutureStatus.fulfilled:
-                return buildAudios(controller.audios.value.toList());
-              case FutureStatus.rejected:
-                return RepeatWidget(controller.loadAudios);
-              default:
-                return const SplashWidget();
-            }
-          },
-        ),
+      child: Observer(
```
Comment (7flash) [inline lib/modules/discover/audios/discover_audios_view.dart:37]: We don't need to completely remove animation here but rather improve it in following way:
- disable complete logo animation
- keep animated opacity

As soon as content loaded, it should show "fadein" content using animated opacity

File: lib/modules/playlist/open_playlist/open_playlist.dart [hunk 2]
```diff
@@ -265,9 +265,11 @@ class _OpenPlaylistPageState
                     .play(controller.playList.audioFiles[index].audioUrlPath);
               },
               child: PlayListAudioTile(
-                audio: widget.playList.audioFiles[index],
-                isPlaying: index == controller.index && controller.isPlaying,
-              ),
+                  audio: widget.playList.audioFiles[index],
+                  isPlaying: index == controller.index && controller.isPlaying,
+                  authorCallback: () {
+                    controller.toggleAudioPlayerMode();
```
Comment (7flash) [inline lib/modules/playlist/open_playlist/open_playlist.dart:271]: why is that necessary?

File: lib/modules/playlist/show_playlists/show_playlists_page.dart [hunk 3]
```diff
@@ -57,10 +54,18 @@ class _ShowPlaylistsPageState
...
+            onTap: () async {
+              MediaOverlays.presentPlayListPlayerAsOverlay(
+                context: context,
+                playList: item,
+              );
+            },
+            child: PlaylistTile(
+              isInEditMode: false,
+              playList: item,
+              onTap: (item) => playlistContextMenu(context, item),
```
Comment (7flash) [inline lib/modules/playlist/show_playlists/show_playlists_page.dart:67]: does it mean both onTap will be triggered here? why?

File: lib/modules/profile/audio/playlist_audio_tile.dart [hunk 4]
```diff
@@ -52,7 +55,15 @@ class PlayListAudioTile extends StatelessWidget {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 mainAxisAlignment: MainAxisAlignment.spaceBetween,
                 children: <Widget>[
-                  Text(audio.userName, style: normal12Hint),
+                  GestureDetector(
+                    onTap: () {
+                      if (authorCallback != null) {
+                        authorCallback();
```
Comment (7flash) [inline lib/modules/profile/audio/playlist_audio_tile.dart:61]: so why is that needed?

File: lib/modules/playlist/open_playlist/open_playlist.dart (hunk 2 above)
Comment (mob-dev) [reply lib/modules/playlist/open_playlist/open_playlist.dart:271]: This was necessary as the author screen was being opened but was hidden behind audio overlay screen. It felt as if nothing has happened on author click, thus I minimized the audio player so that user is able to see the author profile screen.

File: lib/modules/playlist/show_playlists/show_playlists_page.dart (hunk 3 above)
Comment (mob-dev) [reply lib/modules/playlist/show_playlists/show_playlists_page.dart:67]: No, if we click on the three dots button only that method will be triggered. Clicking anywhere else(other than those three dots button) the playlist overlay screen will be opened.

File: lib/modules/profile/audio/playlist_audio_tile.dart (hunk 4 above)
Comment (mob-dev) [reply lib/modules/profile/audio/playlist_audio_tile.dart:61]: The author callback function is required to minimize the playlist overlay, so that user can see the below Author Profile screen. Removing this callback will open the Author Profile screen but it will be behind the Playlist overlay.

File: lib/modules/discover/audios/discover_audios_view.dart (hunk 1 above)
Comment (7flash) [reply lib/modules/discover/audios/discover_audios_view.dart:37]: can you please keep opacity animation and then I will merge it?

Comment (mob-dev) [reply lib/modules/discover/audios/discover_audios_view.dart:56]: I have added the opacity animation in both DiscoverAudio & DiscoverAuthor screens.
